pkgs/build-support/setup-hooks/wrap-gapps-hook/default.nix

```diff
@@ -27,16 +28,22 @@ makeSetupHook {
     # TODO: remove this, packages should depend on GTK explicitly.
     gtk3
 
-    # librsvg provides a module for gdk-pixbuf to allow rendering
-    # SVG icons. Most icon themes are SVG-based and so are some
-    # graphics in GTK (e.g. cross for closing window in window title bar)
-    # so it is pretty much required for applications using GTK.
     librsvg
   ] ++ [
 
     # We use the wrapProgram function.
     makeWrapper
   ];
+
+  # depsTargetTargetPropagated will be buildInputs
+  # the librsvg above should be removed but kept to not break anything that implicitly depended on its binaries
```

Comment:
Maybe move the comment above then. I was quite confused until I noticed it.

Personally, I would just rip the band-aid off. I would not expect this to be very prevalent.

Reply:
> Maybe move the comment above then. I was quite confused until I noticed it.

this comment and the librsvg above it are closer now after rearranging the list

> Personally, I would just rip the band-aid off. I would not expect this to be very prevalent.

Should be a separate PR